packages/peregrine/lib/talons/SignIn/useSignIn.js

```diff
@@ -8,26 +8,20 @@ export const useSignIn = props => {
         setDefaultUsername,
         showCreateAccount,
         showForgotPassword,
-        query
+        signInMutation
     } = props;
 
     const [isSigningIn, setIsSigningIn] = useState(false);
 
     const [, { getCartDetails, removeCart }] = useCartContext();
-    const [
-        { isGettingDetails, getDetailsError },
-        { getUserDetails, setToken }
-    ] = useUserContext();
+    const [, { setToken }] = useUserContext();
 
-    const [signIn, { error: signInError }] = useMutation(query);
+    const [signIn, { error: signInError }] = useMutation(signInMutation);
 
     const errors = [];
     if (signInError) {
         errors.push(signInError.graphQLErrors[0]);
     }
-    if (getDetailsError) {
-        errors.push(getDetailsError);
```

Comment:
You may want to keep this code. You can still propagate the error to the store by calling `getDetails.receive` with an error.